migrate.go

```diff
@@ -143,10 +142,6 @@ func AddNamedMigration(filename string, up func(*sql.Tx) error, down func(*sql.T
 // CollectMigrations returns all the valid looking migration scripts in the
 // migrations folder and go func registry, and key them by version.
 func CollectMigrations(dirpath string, current, target int64) (Migrations, error) {
-	if _, err := os.Stat(dirpath); os.IsNotExist(err) {
```

Comment:
I'm worried that this might enable silent errors if someone misconfigures their directory with actual .sql files.

Let me close this PR and discuss #142 further.